scripts/visualize/visualize.matthew_map.R

```diff
@@ -25,9 +27,16 @@ visualize.matthew_map <- function(viz){
   
   for (i in 1:length(counties)){
     xml_attr(p[[i]], 'id') <- paste0('FIP-', as.character(counties$FIPS[i]))
+    xml_attr(p[[i]], 'class') <- 'county-polygon'
     xml_attr(p[[i]], 'style') <- NULL
     xml_attr(p[[i]], 'clip-path') <- NULL
   }
   
+  g.rivers <- xml_add_child(svg, 'g', id='rivers','class'='river-polyline')
+  pl <- xml_find_all(svg, '//*[local-name()="polyline"]')
+  for (i in 1:length(pl)){
+    xml_add_child(g.rivers, 'polyline', points = xml_attr(pl[i], 'points'))
+  }
+  xml_remove(pl)
```

Comment:
Sounds like you might be able to use the `.copy=FALSE` arg, haven't done it before though